src/safeposix/syscalls/fs_calls.rs

```diff
@@ -1286,14 +1372,17 @@ impl Cage {
                         "we don't support fstat on sockets yet",
                     );
                 }
+                // Streams don't have inodes, so we'll populate statbuf with dummy info
                 Stream(_) => {
                     self._stat_alt_helper(statbuf, STREAMINODE);
                 }
+                // Pipes don't have inodes, so we'll populate statbuf with dummy info
```

Comment:
What does the dummy info signify here?

Reply:
there is no particular reasoning behind using this dummy info, i believe nick mentioned in a previous call that this info was pulled straight from a live environment.